DFE-Digital/get-help-with-remote-education-prototype
# Bring in GOV.UK prototyping toolkit

## What does this PR do?

This PR implements the existing pages in the [GOV.UK Prototype toolkit](https://govuk-prototype-kit.herokuapp.com/docs) to allow us to create protoype pages faster by using the templating tools/macros that already exist.

## How to review this PR?

- Checkout this branch
- Run `npm install` and `npm start` to start the application running on port 3000
- Navigate to `localhost:3000` and check the content of the pages

## What needs to be done following this PR?

The deploy scripts will need to be updated as they currently use the static build pack, however, I don't have access to GOV.UK PaaS so I can't test them to check if they work if I change them :tada:
